cmd/web5/cmd_did.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/tbd54566975/web5-go/dids"
+)
+
+type didResolveCmd struct {
+	URI string `arg:"" name:"uri" help:"The URI to resolve."`
+}
+
+func (c *didResolveCmd) Run(_ context.Context) error {
```

Comment:
You can just omit this argument if it's not used. Kong dynamically injects the types only if they're present.